astropy/modeling/tests/test_models.py

```diff
@@ -516,3 +520,55 @@ def test_voigt_model():
 def test_model_instance_repr():
     m = models.Gaussian1D(1, 2, 3)
     assert repr(m) == '<Gaussian1D(amplitude=1.0, mean=2.0, stddev=3.0)>'
+
+
+@pytest.mark.skipif("not HAS_SCIPY")
+def test_tabular_interp_1d():
+    points = np.arange(0, 5)
+    values = [1., 10, 2, 45, -3]
+    LookupTable = models.tabular_model(1)
+    model = LookupTable(points=points, lookup_table=values)
+    xnew = [0., .7, 1.4, 2.1, 3.9]
+    utils.assert_allclose(model(xnew), [1., 7.3, 6.8, 6.3, 1.8])
+    model = LookupTable(lookup_table=values)
+    utils.assert_allclose(model(xnew), [1., 7.3, 6.8, 6.3, 1.8])
+    with pytest.raises(ValueError):
+        model([0., .7, 1.4, 2.1, 3.9, 4.1])
+    # test extrapolation and fill value
+    model = LookupTable(lookup_table=values, bounds_error=False, fill_value=None)
+    utils.assert_allclose(model([0., .7, 1.4, 2.1, 3.9, 4.1]), [ 1. ,  7.3,  6.8,  6.3,  1.8, -7.8])
+
+
+@pytest.mark.skipif("not HAS_SCIPY")
+def test_tabular_interp_2d():
+    table = np.array([[-0.04614432, -0.02512547, -0.00619557, 0.0144165 , 0.0297525 ],
+       [-0.04510594, -0.03183369, -0.01118008, 0.01201388, 0.02496205],
+       [-0.05464094, -0.02804499, -0.00960086, 0.01134333, 0.02284104],
+       [-0.04879338, -0.02539565, -0.00440462, 0.01795145, 0.02122417],
+       [-0.03637372, -0.01630025, -0.00157902, 0.01649774, 0.01952131]])
+
+    points = (np.arange(0, 5), np.arange(0, 5))
+
+    xnew = np.array([0., .7, 1.4, 2.1, 3.9])
+    LookupTable = models.tabular_model(2)
+    model = LookupTable(points, table)
+    znew = model(xnew, xnew)
+    result = np.array([-0.04614432, -0.03450009, -0.02241028, -0.0069727 ,  0.01938675])
+    utils.assert_allclose(znew, result, atol=10**-7)
+    with pytest.raises(ValueError):
+        model = LookupTable(points=([1.2, 2.3], [1.2, 6.7], [3, 4]))
+
+
+@pytest.mark.skipif("not HAS_SCIPY")
+def test_tabular():
```

Comment:
This contains repetition from `test_tabular_interp_1d()` above. Is it really necessary? Can this become apart of `test_tabular_interp_1d()` instead?